client.go

```diff
@@ -93,31 +90,10 @@ func (c *Client) Production() *Client {
 // transparently before sending the notification. It will return a Response
 // indicating whether the notification was accepted or rejected by the APNs
 // gateway, or an error if something goes wrong.
+//
+// It wraps PushWithCtx for back compatibility.
 func (c *Client) Push(n *Notification) (*Response, error) {
```

Comment:
I think its pretty important to keep this old version around @nathany just so we don't break other projects that depend on it